addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -675,7 +687,10 @@ export class AnalyticDistribution extends Component {
 export const analyticDistribution = {
     component: AnalyticDistribution,
     supportedTypes: ["char", "text"],
-    fieldDependencies: [{ name:"analytic_precision", type: "integer" }],
+    fieldDependencies: [
+        { name: "analytic_precision", type: "integer" },
+        { name: "company_id", type: "many2one" },
```

Comment:
Ah, looks like the tests need to be adapted.
`company_id` needs to be added to the mock model/data

Reply:
Hi @aab-odoo, what are your thoughts on this PR discussion?